e2e/tests/session_steps_test.go

```diff
@@ -336,3 +353,14 @@ func combineEventMatchFns(fns ...func(*abci.Event) bool) func(*abci.Event) bool
 		return true
 	}
 }
+
+// getMsgSubmitProofSenderAddress returns the sender address from the given event.
+func getMsgSubmitProofSenderAddress(event *abci.Event) string {
+	for _, attribute := range event.Attributes {
+		if attribute.Key == "sender" {
+			return attribute.Value
+		}
+	}
+
+	return ""
+}
```

Comment:
Optional, you can do something like this:

```go
import "golang.org/x/exp/slices"

// getMsgSubmitProofSenderAddress returns the sender address from the given event.
func getMsgSubmitProofSenderAddress(event *abci.Event) *string {
	return slices.FindFunc(event.Attributes, func(attribute abci.EventAttribute) bool {
		return attribute.Key == "sender"
	})
```

Reply:
There seem to be no `FindFunc` in slices. I'll be using `slices.IndexFunc`